<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenHelpView</class>
 <widget class="QWidget" name="GenHelpView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="GenKineticScrollWidget" name="kineticArea" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>360</width>
     <height>490</height>
    </rect>
   </property>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>360</width>
      <height>490</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>360</width>
       <height>490</height>
      </rect>
     </property>
     <widget class="QLabel" name="currentImage">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>360</width>
        <height>490</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="GenPageIndicator" name="currentPage" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>450</y>
        <width>360</width>
        <height>40</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(91, 91, 91, 128);</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GenPageIndicator</class>
   <extends>QWidget</extends>
   <header>genpageindicator.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GenKineticScrollWidget</class>
   <extends>QWidget</extends>
   <header>genkineticscrollwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
